.github/workflows/codeql.yml

```diff
@@ -0,0 +1,69 @@
+name: "CodeQL"
```

Comment:
Usually, we specify this as the workflow file name, i.e `codeql`.

Reply:
Removed (dunno, why).